### PR TITLE
LPS-158973 Added z-index value for CKEditor dropdown menus.

### PR DESCRIPTION
**Pull request by fortunatomaldonado**
FWD: https://github.com/fortunatomaldonado/liferay-portal/pull/38

Hello, Let me know if there are any questions or comments about this.
Thank you!

> [LPS-158973](https://issues.liferay.com/browse/LPS-158973) Added z-index value for CKEditor dropdown menus to prevent them from appearing above all other elements on page when scrolling.

### CODE REVIEW

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -253,6 +253,11 @@
 	width: 220px;
 }
 
+.cke_panel.cke_panel,
+.cke_panel.cke_combopanel {
+	z-index: 100 !important;
```
Comment (diegonvs) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/main.scss:258]: Hey @fortunatomaldonado, when taking a look on Clay's Dropdown, I saw they are using `1000` as z-index

image


maybe 100, isn't too little?

**Comment by diegonvs** [comment]
Not related to this PR:

Hey @markocikos, did we intend to move some CKEditor styles to the skin or maybe move skin styles to ckeditor-web?

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (fortunatomaldonado) [reply modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/main.scss:258]: I found `z-index: 900` to work best here. Is this what you mean?

**Review by diegonvs — APPROVED** [review]

**Comment by diegonvs** [comment]
ci:forward

**Comment by markocikos** [comment]
> did we intend to move some CKEditor styles to the skin or maybe move skin styles to ckeditor-web

@diegonvs  Styles are currently split, between `moono-lexicon` theme in CKEditor repo and `*.scss` files ins ckeditor-web. At one point in time, we intended to have a standalone lexicon-based theme, for CKEditor used as a standalone product. Since then, we have moved to more and more DXP-centric vision, with the likes of `senna.js` becoming a part of DXP. Following that vision, I believe that we should, at some point, migrate all styles from `moono-lexicon` to DXP, use default CKEditor theme, and customize styles fully in DXP `*scss` files. This is not set in stone, there are pros ans cons, we need to sync with product folks on this.

**Pull request closed without merging**